src/Core/Package.vala

```diff
@@ -45,6 +45,7 @@ public class AppCenterCore.Package : Object {
     public const string APPCENTER_PACKAGE_ORIGIN = "appcenter";
     private const string ELEMENTARY_STABLE_PACKAGE_ORIGIN = "elementary-stable-jammy-main";
 
+    public PermissionsFlags permissions_flags { get; set; default = PermissionsFlags.UNKNOWN; }
```

Comment:
It isn't necessarily clear in this PR but is there a scenario were the permission *actually* is unknown? Just based on this PR it seems like it makes more sense to default to None but I feel like there very well may be unknowns I am not aware/thinking of